<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>501</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="_videoWindowGroupBox">
        <property name="title">
         <string/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <layout class="QGridLayout" name="_videoWindowGridLayout"/>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Expanding</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="Trackbar" name="_trackbar" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>250</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>501</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOpen_Files">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuRecent_Files">
     <property name="title">
      <string>Recent Files</string>
     </property>
     <addaction name="separator"/>
     <addaction name="separator"/>
    </widget>
    <addaction name="actionOpen_Files"/>
    <addaction name="menuRecent_Files"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionShow_side_by_side"/>
    <addaction name="actionAnchored"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <widget class="QMenu" name="menuAppearance">
     <property name="title">
      <string>Appearance</string>
     </property>
     <addaction name="separator"/>
     <addaction name="actionStandard"/>
     <addaction name="actionGray_Blue"/>
    </widget>
    <addaction name="menuAppearance"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionZoombar"/>
   </widget>
   <addaction name="menuOpen_Files"/>
   <addaction name="menuEdit"/>
   <addaction name="menuTools"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_Files">
   <property name="text">
    <string>Add XML-Track...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="text">
    <string>Options</string>
   </property>
  </action>
  <action name="actionNativ_Open">
   <property name="text">
    <string>Native Open...</string>
   </property>
  </action>
  <action name="actionShow_side_by_side">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>side by side</string>
   </property>
  </action>
  <action name="actionAnchored">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>anchored</string>
   </property>
  </action>
  <action name="actionStandard">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Standard</string>
   </property>
  </action>
  <action name="actionGray_Blue">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Gray Blue</string>
   </property>
  </action>
  <action name="actionZoombar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Zoombar</string>
   </property>
  </action>
  <action name="actionClearRecentFiles">
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Trackbar</class>
   <extends>QWidget</extends>
   <header location="global">gui/trackbar.h</header>
   <container>1</container>
   <slots>
    <signal>timestampChanged(int)</signal>
    <signal>stopClicked()</signal>
    <signal>pauseClicked()</signal>
    <signal>playForwardClicked()</signal>
    <signal>playBackwardClicked()</signal>
    <signal>speedChanged()</signal>
    <signal>finished()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>_trackbar</sender>
   <signal>pauseClicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onPauseClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>298</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_trackbar</sender>
   <signal>stopClicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onStopClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>298</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_trackbar</sender>
   <signal>playForwardClicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onPlayForwardClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>298</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_trackbar</sender>
   <signal>playBackwardClicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onPlayBackwardClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>298</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpen_Files</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onOpenClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onExitClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAbout</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onAboutClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAnchored</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>onActionAnchorToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionShow_side_by_side</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>onActionSideBySideToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionZoombar</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>onActionZoombarToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionStandard</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>useStyleSheet()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionGray_Blue</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>useStyleSheet()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onTimestampChanged(int)</slot>
  <slot>onStopClicked()</slot>
  <slot>onPauseClicked()</slot>
  <slot>onPlayForwardClicked()</slot>
  <slot>onPlayBackwardClicked()</slot>
  <slot>onSpeedChanged(int)</slot>
  <slot>onOpenClicked()</slot>
  <slot>onExitClicked()</slot>
  <slot>onAboutClicked()</slot>
  <slot>onNativeOpenClicked()</slot>
  <slot>onActionAnchorToggled(bool)</slot>
  <slot>onActionSideBySideToggled(bool)</slot>
  <slot>onActionZoombarToggled(bool)</slot>
  <slot>useStyleSheet()</slot>
 </slots>
</ui>
